app/bundles/FormBundle/Translations/en_US/messages.ini

```diff
@@ -126,10 +126,10 @@ mautic.form.form.header.edit="Edit Form - %name%"
 mautic.form.form.header.manualcopy="Manual Copy"
 mautic.form.form.header.new="New Form"
 mautic.form.form.header.preview="Preview Form"
-mautic.form.form.help.automaticcopy="With this option, Mautic will automatically generate the form where you include this code and thus it will always be the most up-to-date version."
-mautic.form.form.help.automaticcopy.js="Via Javascript (recommended)"
-mautic.form.form.help.automaticcopy.iframe="Via iframe"
-mautic.form.form.help.automaticcopy.iframe.note="Note: Adjust the width and height attribute so the form fits in."
+mautic.form.form.help.embedded="When you use this option, the entire form can be edited within the platform and these changes will automatically be reflected on any page where it is inserted."
+mautic.form.form.help.embedded.js="Via Javascript (recommended)"
```

Comment:
Do we need to change the translation keys for translations that aren't changing? This way all the translations in all languages will have to translate this again. Seems like a lot of work for changing a translation key that no one actually sees.

Reply:
Thanks for pointing!